src/domain/challenge/challenge/challenge.service.ts

```diff
@@ -82,7 +84,7 @@ export class ChallengeService {
   async createChallenge(
     challengeData: CreateChallengeInput,
     hubID: string,
-    agentInfo?: AgentInfo
+    agentInfo: AgentInfo
```

Comment:
I am not sure why AgentInfo is optional here, but this change can potentially cause a regression.

Reply:
Reverted, it was for an earlier version of the code.